nixos/modules/services/databases/influxdb2.nix

```diff
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.influxdb2;
+  configFile = pkgs.writeText "config.json" (builtins.toJSON cfg.settings);
+in
+{
+  options = {
+    services.influxdb2 = {
+      enable = mkEnableOption "the influxdb2 server";
+      package = mkOption {
+        default = pkgs.influxdb2;
+        defaultText = "pkgs.influxdb2";
+        description = "influxdb2 derivation to use";
+        type = types.package;
+      };
+      settings = mkOption {
+        default = { };
+        description = "configuration options for influxdb2";
```

Comment:
As above, should end in period. Also can you please link a description to accepted values? `man` page for the conf file, or whatever.